src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1559,6 +1562,69 @@ public String toVariableNameString(EnchantmentOffer eo) {
 			.name("Experience Cooldown Change Reason")
 			.description("Represents a change reason of an <a href='events.html#experience cooldown change event'>experience cooldown change event</a>.")
 			.since("INSERT VERSION"));
+
+		Classes.registerClass(new RegistryClassInfo<>(LootTables.class, Registry.LOOT_TABLES, "loottabletype", "loot table types")
+				.user("(loot )?table ?types?")
```

Comment:
```suggestion
				.user("(loot ?)?table ?types?")
```
Should keep the same pattern throughout it, meaning `loot tabletype` and `loottabletype` should work